src/views/Predictions/components/RoundCard/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
```

Comment:
This is the controller component that determines what kind of round card to show.